liquidity/ui/src/PerpsCommitOrder.tsx

```diff
@@ -49,14 +49,16 @@ export function PerpsCommitOrder() {
   });
 
   const { data: settlementStrategy } = usePerpsGetSettlementStrategy({
+    provider,
+    perpsMarketId: params.market,
```

Comment:
```suggestion
    perpsMarketId: params.perpsMarketId,
```
I reckon it is worth renaming params too. For consistency and to avoid any collisions

Same as it was done for `perpsAccountId`

Reply:
Rename `params.market` => `params.perpsMarketId`